src/selectors/entities/channel_categories.ts

```diff
@@ -248,7 +248,12 @@ export function makeFilterAutoclosedDMs(): (state: GlobalState, channels: Channe
                 remaining = limitPref;
             }
 
+            const currentChannel = filtered.find((channel) => channel.id === currentChannelId);
```

Comment:
Instead of looking through filtered, I think it would be better to pass the current channel into this instead of the current channel ID. That way, we don't need to null check `currentChannel`

Reply:
In that case we still need to add another selector and get the channel as an extra step, IMO I think this fits more with the flow